pkg/crinit/standalone/standalone_test.go

```diff
@@ -177,7 +210,7 @@ func TestCreateNamespace(t *testing.T) {
 	t.Run("simple namespace creation", func(t *testing.T) {
 		name := "test"
 		client := fake.NewSimpleClientset()
-		ns, err := createNamespace(client, name, false)
+		ns, err := util.CreateNamespace(client, name, false)
```

Comment:
This seems like it should be moved into a util_test rather than here in standalone_test.

Reply:
Yes, when I follow up with an aggregated_test I'll consolidate for sure.